Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -6,6 +6,7 @@
 #include "Subsystems/WorldSubsystem.h"
 #include "PoolManagerTypes.h"
 #include "Materials/MaterialInstanceDynamic.h"
```

Comment:
Is not really related to this review, but seems I missed it before: please move `#include "Materials/MaterialInstanceDynamic.h"` to .cpp :D

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/f060e0407563a24376d6bebc305ac285e17c727b